.hashibot.hcl

```diff
@@ -488,6 +488,7 @@ behavior "regexp_issue_labeler_v2" "service_labels" {
     ],
     "service/s3" = [
       "aws_canonical_user_id",
+      "aws_awslogsdelivery_canonical_user_id",
```

Comment:
[ask]: Is this datasource name better? 'aws_cloudfront_canonical_user_id' ?

I'd like to solve issue https://github.com/terraform-providers/terraform-provider-aws/issues/12512 .

Following https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/AccessLogs.html, we should add the canonical user for awslogsdelivery.

So, I chose this name because I don't think it's strictly aws_cloudfront_canonical_user_id.

Reply:
I changed to cloudfront_canonical_user_id because there are many canonical_ids for awslogsdelivery, not only for cloudfront